packages/arb-token-bridge-ui/src/util/withdrawals/fetchTokenWithdrawalsFromEventLogsSequentially.ts

```diff
@@ -100,22 +81,9 @@ export async function fetchTokenWithdrawalsFromEventLogsSequentially({
     })
   }
 
-  const gateways = await getGateways(provider)
-  const senderNonce = await backOff(() => getNonce(sender, { provider }))
-
-  // sender queries; only add if nonce > 0
-  if (senderNonce > 0) {
-    addQuery(buildQueryParams({ sender, gateways: [gateways.standardGateway] }))
-    addQuery(buildQueryParams({ sender, gateways: [gateways.wethGateway] }))
-    addQuery(buildQueryParams({ sender, gateways: [gateways.customGateway] }))
-    addQuery(buildQueryParams({ sender, gateways: gateways.otherGateways }))
-  }
-
-  // receiver queries
-  addQuery(buildQueryParams({ receiver, gateways: [gateways.standardGateway] }))
-  addQuery(buildQueryParams({ receiver, gateways: [gateways.wethGateway] }))
-  addQuery(buildQueryParams({ receiver, gateways: [gateways.customGateway] }))
-  addQuery(buildQueryParams({ receiver, gateways: gateways.otherGateways }))
```

Comment:
baiscally, all this was moved to `fetchWithdrawals` so you can have some more control from the outside